identity.html

```diff
@@ -902,7 +931,317 @@ <h2><dfn>RTCIdentityAssertion</dfn> Attributes</h2>
+        <code>setRemoteDescription</code> MUST NOT await the completion
+        of identity validation, and whether
+        <a href="#dfn-validate-the-identity">identity validation</a> fails
+        or not MUST NOT influence whether the promise returned by
+        <code><a>setRemoteDescription</a></code> is rejected.</p>
+      </section>
+    </section>
+    <section>
+      <h4>RTCError Interface Extensions</h4>
+      <div>
+        <pre class="idl"
+>partial interface RTCError {
+  readonly attribute long? httpRequestStatusCode;
+};</pre>
+      </div>
+      <section>
+        <h2>Attributes</h2>
+        <dl data-link-for="RTCError" data-dfn-for="RTCError" class="attributes">
+          <dt data-tests="RTCError.html"><dfn data-idl><code>httpRequestStatusCode</code></dfn> of type
+          <span class="idlAttrType">long</span>, readonly,
+          nullable</dt>
+          <dd>
+            <p>If <code>errorDetail</code> is <code>"idp-load-failure"</code>
+            this is the HTTP status code of the IdP URI response.</p>
+          </dd>
+        </dl>
+      </section>
+      <h3><dfn>RTCErrorInit</dfn> Dictionary</h3>
+      <div>
+        <pre class="idl"
+>partial dictionary RTCErrorInit {
+  long httpRequestStatusCode;
+};</pre>
+        <p data-dfn-for="RTCErrorInit">The <dfn data-idl>httpRequestStatusCode</dfn> member of <code>RTCErrorInit</code> has the same definition as the attribute of the same name of <a>RTCError</a>.</p>
       </div>
+      <section>
+        <h2>Dictionary <a>RTCError</a> Members</h2>
+        <dl data-link-for="RTCError" data-dfn-for="RTCError"
+        class="dictionary-members">
+          <dt><code>httpRequestStatusCode</code> of type
+          <span class="idlMemberType">long</span></dt>
+          <dd>
+            <p>See <code><a>RTCError</a></code>'s
+            <code>httpRequestStatusCode</code>.</p>
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h3><code><dfn>RTCErrorDetailTypeIdp</dfn></code> Enum</h3>
+        <div>
+          <pre class="idl"
+>enum RTCErrorDetailTypeIdp {
+  "data-channel-failure",
+  "dtls-failure",
+  "fingerprint-failure",
+  "sctp-failure",
+  "sdp-syntax-error",
+  "hardware-encoder-not-available",
+  "hardware-encoder-error"
+  "idp-bad-script-failure",
+  "idp-execution-failure",
+  "idp-load-failure",
+  "idp-need-login",
+  "idp-timeout",
+  "idp-tls-failure",
+  "idp-token-expired",
+  "idp-token-invalid",
+};</pre>
+        </div>
+        <p>Only the error detail types whose names are prefixed with "idp-" are
+        defined by this specification. They are to be considered extensions of
+        the <a href="https://w3c.github.io/webrtc-pc/#dom-rtcerrordetailtype">
+        RTCErrorDetailType</a> enum. The rest are here because a WebIDL enum
+        must be described in one place only. This can be simplified should
+        partial enum support materialize.</p>
```

Comment:
Given that WebIDL restricts us from describing what we want to describe anyway (i.e. the enum "RTCErrorDetailTypeIdp" is not actually a new enum type that is to be used), and we have to clarify how this is to be interpreted in a paragraph, might I suggest we don't list the old enum values, just the new ones?

And perhaps we can add a comment in WebIDL e.g. "// This is an extension of RTCErrorDetailType from [[WEBRTC-PC]]"?

Alternatively, if we are to say that this is the new enum to replace the old and it is a complete list of values, then maybe we can get away with calling it the old name "RTCErrorDetailType". However I don't like this because then it is "impossible" to implement multiple different extension specs that add different values to it.

But whatever, as long as the reader understands what we mean.

Reply:
I chose adding the comment.  Using the same name will likely produce errors here once respec is updated in this repo (why some of the WebIDL is shown on white instead of blue btw).

Reply:
LGTM